operator/v1/types_machineconfiguration.go

```diff
@@ -156,3 +169,116 @@ const (
 	// This feature only supports MAPI machinesets at this time.
 	MachineAPI MachineManagerMachineSetsAPIGroupType = "machine.openshift.io"
 )
+
+type NodeDisruptionPolicySpec struct {
+	// userPolicies define user-provided node disruption policies
+	// +optional
+	UserPolicies []NodeDisruptionPolicyConfig `json:"userPolicies"`
+
+	// clusterDefaultPolicies is managed by the Machine Config Operator, and reflects the latest cluster defaults
+	// +optional
+	ClusterDefaultPolicies []NodeDisruptionPolicyConfig `json:"clusterDefaultPolicies"`
+}
+
+type NodeDisruptionPolicyStatus struct {
+	// clusterPolicies is a merge of cluster default and user provided node disruption policies.
+	// +optional
+	ClusterPolicies []NodeDisruptionPolicyConfig `json:"clusterPolicies"`
```

Comment:
status should use a different type because you're likely to grow different fields

Reply:
done